src/parser/mod.rs

```diff
@@ -7379,6 +7380,31 @@ impl<'a> Parser<'a> {
         })
     }
 
+    /// Parse an REPLACE statement
+    pub fn parse_replace(&mut self) -> Result<Statement, ParserError> {
+        if !dialect_of!(self is MySqlDialect) {
```

Comment:
Can you please also support Generic dialect, as explained in https://github.com/sqlparser-rs/sqlparser-rs?tab=readme-ov-file#new-syntax

Reply:
Added generic dialects